cmd/ceremony/cert.go

```diff
@@ -243,15 +266,12 @@ func makeTemplate(randReader io.Reader, profile *certProfile, pubKey []byte, ct
 		return nil, errors.New("at least one key usage must be set")
 	}
 
+	templateSubject := profile.Subject()
```

Comment:
Inline the function call on line 274, since this variable name isn't used anywhere else.

Reply:
Addressed in new PR #5326